OpenRA.Game/Actor.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace OpenRA
 {
-	public sealed class Actor : IScriptBindable, IScriptNotifyBind, ILuaTableBinding, ILuaEqualityBinding, ILuaToStringBinding, IEquatable<Actor>, IDisposable
+	public sealed class Actor : IScriptBindable, IScriptNotifyBind, ILuaTableBinding, ILuaEqualityBinding, ILuaToStringBinding, ISync, IEquatable<Actor>, IDisposable
```

Comment:
I'm pretty sure we don't sync-test Actors (only traits), so this isn't very useful.